checker/check_result.go

```diff
@@ -139,18 +139,35 @@ type SecurityPolicyData struct {
 	Files []File
 }
 
+// ToolConfigFile represents a config file
+// for a tool.
+type ToolConfigFile struct {
+	Name string
+	URL  string
+	Desc string
```

Comment:
URL/Desc -> trying this out. Could be a nice way to embed more description in results. wdut?